src/lib/third_party/src/ndpi_patricia.c

```diff
@@ -238,6 +238,16 @@ static ndpi_prefix_t * ndpi_New_Prefix2 (int family, void *dest, int bitlen, ndp
       }
       memcpy (&prefix->add.sin, dest, sizeof(struct in_addr));
     }
+    else if(family == AF_MAC) {
```

Comment:
Is it an apple exclusive address family?
And what's the purpose of that AF?

Reply:
I *think* that the goal is to have MAC addresses into the tree, in the same way we have ipv4 or ipv6 addresses
The constant `AF_MAC` is added for "compatibility" with `AF_INET` and `AF_INET6`, so that you can specify the tree "type" in the same way.
Note that I am guessing here... never use mac addresses and it seems `AF_MAC` is in nDPI pretty much since forever

Reply:
ty for clarification <3